_live-tests/main_test.go

```diff
@@ -0,0 +1,32 @@
+package live_tests
+
+import (
+	"log"
+	"testing"
+
+	"os"
+
+	"github.com/appoptics/appoptics-api-go"
+)
+
+const (
+	TestPrefix        = "iNtEgRaTiOnTEST"
+	CreatedNameString = "created"
+	UpdatedNameString = "updated"
+)
+
+var (
+	client *appoptics.Client
+)
+
+func TestMain(m *testing.M) {
+	token := os.Getenv("APPOPTICS_TOKEN")
+	if token == "" {
+		log.Fatal("set APPOPTICS_TOKEN in the environment")
+	}
+
+	client = appoptics.NewClient(token)
+	// Uncomment the below to see response status/body stdout while tests run
+	//client = appoptics.NewClient(token, appoptics.SetDebugMode())
```

Comment:
Maybe this should be configurable via ENV, and be enabled for CI? Or is that too verbose?

Reply:
Yeah I think that's a little too verbose -- it dumps entire request bodies for each request, which can make it hard to read. I'd vote against making it the default, but I'm not against an available ENV setting for it.